### PR TITLE
Core Elements documentation 

### PR DESCRIPTION
**Pull request by FabienLelaquais**
- Added specific documentation for the scenario_selector, scenario, scenario_dag and data_node_selector controls.

- Remove TODOs from the Core Elements home page and RelNotes.
- run-time => runtime everywhere

### CODE REVIEW

File: docs/manuals/gui/corelements/scenario.md_template [hunk 1]
```diff
@@ -1,7 +1,42 @@
...
+the information for that scenario.
+
+Here is what the control would look like when representing a scenario entity:
+<figure class="tp-center">
+  <img src="../scenario-init-d.png" class="visible-dark"  width="60%"/>
+  <img src="../scenario-init-l.png" class="visible-light" width="60%"/>
+  <figcaption>Viewing the parameters of a scenario</figcaption>
+</figure>
+
+Next the scenario's label at the top of the control, the 'fly' button can trigger the scenario
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/corelements/scenario.md_template]: fly button ? or play button

File: docs/manuals/gui/corelements/scenario.md_template [hunk 2]
```diff
@@ -1,7 +1,42 @@
...
+
+Users can also add or modify custom properties:
+<figure class="tp-center">
+  <img src="../scenario-edit-props-d.png" class="visible-dark"  width="60%"/>
+  <img src="../scenario-edit-props-l.png" class="visible-light" width="60%"/>
+  <figcaption>Editing custom properties</figcaption>
+</figure>
+
+Clicking the "New Property Key" line will create a new property.<br/>
+Clicking a custom property value field will let users change the property value.</br>
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/corelements/scenario.md_template]: or key

File: docs/manuals/gui/corelements/scenario_selector.md_template [hunk 3]
```diff
@@ -1,8 +1,77 @@
...
+behavior, controlled by the [*show_add_button*](#p-show_add_button) property, is to display a
+button letting users create new scenarios:
+
+<figure class="tp-center">
+  <img src="../scenario_selector-empty-d.png" class="visible-dark"  width="80%"/>
+  <img src="../scenario_selector-empty-l.png" class="visible-light" width="80%"/>
+  <figcaption>No scenario to select</figcaption>
+</figure>
+
+When the user presses that button, a form appears so the settings of the new scenario can be set:
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/corelements/scenario_selector.md_template]: so that the ?

**Review by FredLL-Avaiga — APPROVED** [review]

File: docs/manuals/gui/corelements/scenario.md_template (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/corelements/scenario.md_template]: Play button it is

File: docs/manuals/gui/corelements/scenario.md_template (hunk 2 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/corelements/scenario.md_template]: Done.

File: docs/manuals/gui/corelements/scenario_selector.md_template (hunk 3 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/corelements/scenario_selector.md_template]: Better. Thanks.

**Comment by jrobinAV** [comment]
@FabienLelaquais Can we delete the branch ?

**Pull request merged**